quirks/password-rules.json

```diff
@@ -218,6 +218,9 @@
     "gmx.net": {
         "password-rules": "minlength: 8; maxlength: 40; allowed: lower, upper, digit, [-<=>~!|()@#{}$%,.?^'&*_+`:;\"[]];"
     },
+    "google.com": {
+        "password-rules": "minlength: 8; allowed: lower, upper, digit, [-!\"#$%&'()*+,./:;<=>?@[^_{|}~]];"
```

Comment:
Are you attempting to call out any ascii-printable character here that Google does not allow?

Reply:
I’m guessing space?

Reply:
@rmondello correct spaces are not allowed hence the specific set of special characters are included.